### PR TITLE
Search component

### PR DESCRIPTION
**Pull request by soyyogi**

### CODE REVIEW

**Review by VictorGreco — CHANGES_REQUESTED** [review]
Better remove the `.eslintcache` file from this commit.

File: src/components/Dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Dropdown.css';
+
+function Dropdown({ options }: { options: string[] }): JSX.Element {
```
Comment (VictorGreco) [inline src/components/Dropdown.tsx:4]:
```suggestion
interface DropDownPops {
    options: String[];
}
function Dropdown({ options }: DropDownPops): JSX.Element {
```

File: src/components/Dropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    const optionsList = options.map((option: string, index: number): JSX.Element => {
+        return <option key={index} value={option.toLowerCase()}>{option}</option>
+    })
```
Comment (VictorGreco) [inline src/components/Dropdown.tsx:7]: missing `;`

File: src/components/Dropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+    return (
+        <div className="dropdown">
+            <select>{optionsList}</select>
+        </div>
+    )
```
Comment (VictorGreco) [inline src/components/Dropdown.tsx:13]: missing `;`

File: src/components/FilterCard.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+function FilterCard({ title, children}: { title: string; children: JSX.Element}): JSX.Element {
+    return (
+        <div className="filterCard">
+            <div className="filterCard_checkbox">
+                <Input type="checkbox" placeholder=""/>
+            </div>
+            <p className="filterCard_title">{title}</p>
+            {children}
+        </div>
+    )
```
Comment (VictorGreco) [inline src/components/FilterCard.tsx:14]: missing `;`

File: src/components/Input.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Input.css';
+
+function Input({ type, placeholder, min, max }: { type: string; placeholder: string; min?: number; max?: number }): JSX.Element {
```
Comment (VictorGreco) [inline src/components/Input.tsx:4]:
```suggestion

interface InputProps {
    type: string;
    placeholder: string;
    min?: number;
    max?: number;
}

function Input({ type, placeholder, min, max }: InputProps): JSX.Element {
```

File: src/components/Input.tsx [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+    return (
+        <div className="input">
+            <input type={type} placeholder={placeholder} min={min} max={max} />
+        </div>
+    )
```
Comment (VictorGreco) [inline src/components/Input.tsx:9]: missing `;`

File: src/components/FilterCard.tsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './FilterCard.css';
+import Input from './Input';
+
+function FilterCard({ title, children}: { title: string; children: JSX.Element}): JSX.Element {
```
Comment (VictorGreco) [inline src/components/FilterCard.tsx:5]:
```suggestion

interface FilterCardProps {
    title: string;
    children: JSX.Element;
}

function FilterCard({ title, children}: FilterCardProps): JSX.Element {
```

**Review by VictorGreco — CHANGES_REQUESTED** [review]
I like the code so much, just some missing `;` and it's better to create an interface for Components types so you can later export it if needed.

**Comment by soyyogi** [comment]
I didn't put interface before because i had to implement more props for events and now while creating event listeners i used interfaces

**Comment by VictorGreco** [comment]
@soyyogi as talked on other channels you can declare multiple interfaces just by doing:

```
interface foo1 {

}

interface foo2 {

}

//some other code

```

**Pull request closed without merging**